style.css

```diff
@@ -0,0 +1,103 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  font-family: "Poppins", sans-serif;
+  box-sizing: border-box;
+}
+
+body {
+  width: 375px;
+  height: 718px;
+  margin: auto;
+}
```

Comment:
- [ OPTIONAL] Kindly note that you are using a fixed width for your project. I suggest you give it a width of 100% so that it accommodates more devices. Currently its only accommodating `Galaxy note ii`